package.json

```diff
@@ -12,17 +11,20 @@
     "gatsby-plugin-react-helmet": "^3.0.5",
     "gatsby-plugin-remove-serviceworker": "^1.0.0",
     "gatsby-plugin-sass": "^2.1.14",
-    "gatsby-plugin-sharp": "^2.0.17",
+    "gatsby-plugin-sharp": "^2.4.5",
     "gatsby-plugin-sitemap": "^2.0.4",
     "gatsby-plugin-twitter": "^2.0.8",
     "gatsby-remark-external-links": "^0.0.4",
     "gatsby-remark-images-medium-zoom": "^1.3.0",
     "gatsby-remark-prismjs": "^3.2.0",
     "gatsby-remark-relative-images": "^0.2.1",
+    "gatsby-source-meetup": "^1.1.0",
     "gatsby-transformer-sharp": "^2.2.14",
     "gatsby-transformer-yaml": "^2.2.9",
     "h5o": "^0.11.3",
     "jsdom": "^11.11.0",
+    "moment": "^2.24.0",
```

Comment:
https://github.com/you-dont-need/You-Dont-Need-Momentjs

Worth a look !

Reply:
Done, I replaced it by [dayjs](https://github.com/iamkun/dayjs). Thanks !